pageserver/src/tenant.rs

```diff
@@ -4004,6 +3969,13 @@ mod tests {
                 .create_test_timeline(TIMELINE_ID, Lsn(0x7000), DEFAULT_PG_VERSION, &ctx)
                 .await?;
             make_some_layers(tline.as_ref(), Lsn(0x8000)).await?;
+            // so that all uploads finish & we can call harness.load() below again
+            tenant
+                .shutdown(Default::default(), true)
+                .instrument(info_span!("test_shutdown", tenant_id=%tenant.tenant_id))
+                .await
+                .ok()
+                .unwrap();
```

Comment:
Just noticed these are using the static `TIMELINE_ID`. Together with global task_mgr, we should see some confluence as the tests are running in randomized order.

I'll push a re-run of the tests soon, but unsure if we need to fix right away..

Reply:
This wasn't the cause for the test failures.

The reason for the failures was: https://github.com/neondatabase/neon/pull/5177